File: AppointmentSchedulerAPI/layers/ServiceLayer/v1/Controllers/DTO/Response/ScheduledService.cs
namespace AppointmentSchedulerAPI.layers.ServiceLayer.v1.Controllers.DTO.Response
{
    public class ScheduledServiceDTO
    {
        public DateTime? EndDate { get; set; }
        public DateTime? StartDate { get; set; }
        public double? Price { get; set; }
        public int? Minutes { get; set; }
        public string? Name { get; set; }
        public Guid Uuid { get; set; }
    }
}

File: AppointmentSchedulerWeb/src/view-model/business-entities/scheduled-service.ts
export class ScheduledService {
  name: string;
  price: number;
  minutes: number;
  uuid: string;
  startDate: Date;
  endDate: Date;

  constructor(name: string, price: number, minutes: number, uuid: string, startDate: Date, endDate: Date) {
    this.name = name;
    this.price = price;
    this.minutes = minutes;
    this.uuid = uuid;
    this.startDate = startDate;
    this.endDate = endDate;
  }
}
